docs/chapters/templates/yocto-templates.rst

```diff
@@ -0,0 +1,29 @@
+Yocto Project integration
+=========================
+
+This section describes how to include a shared library or platform service in the target image using the `Yocto Project`_. A key part of this is an open source build system, based around the OpenEmbedded architecture, that enables developers to create their own Linux distribution specific to their environment. BitBake_ is the build tool used in this setup:
+
+    BitBake executes tasks according to provided metadata that builds up the tasks. Metadata is stored in recipe (.bb) and related recipe "append" (.bbappend) files, configuration (.conf) and underlying include (.inc) files, and in class (.bbclass) files. The metadata provides BitBake with instructions on what tasks to run and the dependencies between those tasks.
```

Comment:
I prefer to put double backticks around file extensions - but I'll leave the decision to you :)